thirdparty/djvulibre/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ ExternalProject_Add(
     DOWNLOAD_COMMAND ${CMAKE_COMMAND} -P ${GIT_CLONE_SCRIPT_FILENAME}
     PATCH_COMMAND NOCONFIGURE=1 ./autogen.sh
     BUILD_IN_SOURCE 1
-    CONFIGURE_COMMAND sh -c ${CFG_CMD_STR} COMMAND sed -i "s|-lstdc++||g" libtool
+    CONFIGURE_COMMAND sh -c ${CFG_CMD_STR} COMMAND sed -ie "s|-lstdc++||g" libtool
```

Comment:
Does this work properly?

I remember some weird crap about that when tweaking soem KindleTool scripts on OS X.

Right now they still read sed -e expr -i '' FILE

(Note the empty quotes, which, IIRC, controlled the backup extension).